demo/index.js

```diff
@@ -326,7 +328,14 @@ $(function() {
             session.terminate();
         });
 
-        session.on('accepted', function() { console.log('Event: Accepted'); });
+        session.on('active-call', function () {
```

Comment:
Instead of this create a new option `enableQos` in WebPhone `options` and in both places where you had `session.emit('active-call', ...)` replace the emit with `session.startQosStatsCollection()`.